examples/gno.land/p/demo/avl/tree.gno

```diff
@@ -43,11 +43,39 @@ func (tree *Tree) Remove(key string) (value interface{}, removed bool) {
 }
 
 // Shortcut for TraverseInRange.
-func (tree *Tree) Iterate(start, end string, cb func(*Node) bool) bool {
-	return tree.node.TraverseInRange(start, end, true, true, cb)
+func (tree *Tree) Iterate(start, end string, cb IterCbFn) bool {
```

Comment:
I came across this PR. It looks like a lot of this is [already implemented](https://github.com/gnolang/gno/blob/ccc6d5b789bbb6a267135b05047db8b85adc74c7/examples/gno.land/p/demo/avl/tree.gno#L64-L97). Perhaps this PR is no longer necessary?